### ISSUE TITLE
Visible Editor Window

### ISSUE THREAD

**Issue by ashea-code** [ufna/VaQuoleUI#19]
Something I came across while working with this awesome plugin, I minimized the editor after stopping a PIE session using a UI component with site Blackle. And I saw this...

![Editor Window](http://puu.sh/bKbOK/48d7507e4a.jpg)

**Comment by ufna** [comment]
It's interesting. Are you using multi-monitor system?

**Comment by ashea-code** [comment]
Not at all, it seems that adding a few lines to the Qt source code fixed it for me
